gradle.properties

```diff
@@ -1,8 +1,8 @@
 group = org.jetbrains.kotlinx
-version = 0.1.0
+version = 0.2.0
 kotlin.code.style=official
 
-kotlin_version=1.3.40-eap-104
+kotlinVersion=1.3.60-eap-76
 
 kotlin.incremental.multiplatform=true
-org.jetbrains.kotlin.native.jvmArgs=-XX:TieredStopAtLevel=1
+kotlin.native.jvmArgs=-Xmx6G
```

Comment:
I wonder why such setting is required for compiling such a small library like kotlinx.cli